Makefile

```diff
@@ -109,7 +109,7 @@ BUILD_TARGETS := build install
 
 build: BUILD_ARGS=-o $(BUILDDIR)/
 build-linux:
-	GOOS=linux GOARCH=amd64 LEDGER_ENABLED=false $(MAKE) build
+	GOOS=linux GOARCH=$(if $(findstring aarch64,$(shell uname -m)),arm64,amd64) LEDGER_ENABLED=false $(MAKE) build
```

Comment:
```suggestion
	GOOS=linux GOARCH=$(if $(findstring aarch64,$(shell uname -m)) || $(findstring arm64, $(shell uname -m)),arm64,amd64) LEDGER_ENABLED=false $(MAKE) build
```

Reply:
Because `uname -m` reports differently on Debian/M1 vs. MacOS/M1:

```
johnk@debian:~/src/cosmos-sdk$ uname -m
aarch64
```

MacOS/M1
```
frumiousj@dirtycomputer cosmos-sdk % uname -m
arm64
frumiousj@dirtycomputer cosmos-sdk % uname
Darwin
```